contents/uix/uix.md

```diff
@@ -0,0 +1,47 @@
+# UI/UX
+
+Author: [Ang Shi Ya](https://github.com/AngShiYa)
+
+## Overview
+
+User Interface (UI) and User Experience (UX) are terms often used interchangeably. In fact, These two terms are quickly becoming dangerous buzzwords that are used inaccurately, creating massive confusion in the industry. A simple google search will show that UI Designer and UX Designer jobs are listed under similar titles and descriptions. 
+
+Learning the key differences between UI and UX can not only help us identify and settle down for the correct job role, but can also help us apply each of these concepts appropriately to our own applications. This document will address the difference between UI and UX as well as some of the techniques used in UI Design and UX Design.
+
+## User Interface vs User Experience
+
+UI is about the design of the buttons, the layout of the website and the responsiveness of the application. It encompasses the presentation and interactivity of the application. On the other hand, UX is concern with the ease of use and the pleasure provided through using the application. In short, UI is about the look and feel while UX is about customer satisfaction.
+
+Due to the overlapping nature of UI and UX, you might still have some questions such as *'Doesn't improving the UI also improve the UX?'* Indeed, it does. In fact, UI **can** be part of UX.
+
+> Note that I used **can** because UI is not longer a necessity to UX due to the emergence of [zero UI](http://blog.careerfoundry.com/ui-design/what-is-zero-ui), which won't be covered here.
+
+To achieve a better understanding, let's look at real life examples of good UI but bad UX. A prime example would be the [Mystery Meat Navigation](https://www.techinasia.com/talk/material-design-mystery-meat-navigation-problem) problem. It refers to buttons and links that do not explain themselves; users have to click on them to find out where it leads. In essence, the pursuit of clean and neat UI has resulted in omission of details that provide clarity for the user. This results in bad UX because people do not like to puzzle over how things work. A good UX should never make the user think.
```

Comment:
"To gain..." seems more appropriate here. Also check out Norman doors for bad UX